packages/domain-picker/src/info-tooltip/index.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, useState } from 'react';
+import classnames from 'classnames';
+import { Popover } from '@wordpress/components';
+import { Icon, info } from '@wordpress/icons';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	children: React.ReactNode;
+	className?: string;
+	id?: string;
+	position?:
+		| 'top left'
+		| 'top right'
+		| 'top center'
+		| 'middle left'
+		| 'middle right'
+		| 'middle center'
+		| 'bottom left'
+		| 'bottom right'
+		| 'bottom center';
+	noArrow?: boolean;
+}
+
+const InfoTooltip: FunctionComponent< Props > = ( {
+	children,
+	className,
+	id,
+	position = 'bottom center',
+	noArrow = true,
+} ) => {
+	const [ showTooltip, setShowTooltip ] = useState< boolean >( false );
+	const handleClick = () => {
+		setShowTooltip( ! showTooltip );
+	};
+	const handleClose = () => {
+		setShowTooltip( false );
+	};
+
+	return (
+		<>
+			<button
+				type="button"
+				aria-haspopup
+				data-testid="info-tooltip"
+				onClick={ handleClick }
+				className={ classnames( 'info-tooltip', 'domain-picker__info-tooltip', className, {
+					'is-active': showTooltip,
+				} ) }
+			>
+				<Icon icon={ info } />
+				{ showTooltip && (
+					<Popover
+						id={ id }
+						className={ classnames( 'info-tooltip__content', className ) }
```

Comment:
`className` us un-used for both this and `button`? I don't have strong preference, but I think I'd just drop it here until it's actually used.

Reply:
On second think, it's prolly to make mimick more the Calypso counterpart of this component to possibly make it re-usable later?

Reply:
I made use of the `className` prop as I had the class `domain-picker__info-tooltip` hard coded within the `<InfoTooltip />` component. This has now been passed down through the `className` prop for flexibility.